sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -805,6 +810,7 @@ RIGHT: 'RIGHT';
 FULL: 'FULL';
 NATURAL: 'NATURAL';
 ON: 'ON';
+PIVOT: 'PIVOT';
```

Comment:
Could you add the keywords you added here to `nonReserved` (line 723)? Also update the suite `TableIdentifierParserSuite`?

Reply:
Sure, I'll update `TableIndentifierParserSuite`. I believe I've added them to `nonReserved` already. Did I miss something?